docs/api/verify-service-provider-with-matching-api.swagger.yml

```
@@ -196,14 +199,14 @@ components:
             - NO_MATCH
           description: Returned when the user has not been successfully matched to your local datastore
     CancellationResponseBody:
-      description: Returned when the user has cancelled 
+      description: Returned when the user has cancelled
       type: object
       properties:
         scenario:
           type: string
           enum:
             - CANCELLATION
-          description: Returned when the user has cancelled 
+          description: Returned when the user has cancelled
```

We can use the SAML status code here too.

---

@chrisholmes can you check I got the right status code here:

```suggestion
          description: Returned when the user has cancelled.  The SAML response for the GOV.UK Verify Hub contains the `urn:oasis:names:tc:SAML:2.0:status:Requester` status code.

```